dictionaries/dict.json

```diff
@@ -127891,12 +127890,13 @@
 "TPHOEB/HRESZ/OEB/HRAOEPBLG": "noblesse oblige",
 "TPHOEB/HRESZ/PWHRAOEPBLG": "noblesse oblige",
 "TPHOEB/HREU": "nobly",
-"TPHOEBD": "antibody",
+"TPHOEBD": "nobody",
 "TPHOEBD/-S": "antibodies",
 "TPHOEBD/THAS": "antibodies that",
 "TPHOEBDZ": "antibodies",
 "TPHOEBG": "notebook",
 "TPHOEBG/KEU": "gnocchi",
+"TPHOEBGDZ": "antibody",
```

Comment:
I'm not super sure about this one @paulfioravanti . Is it worth moving this to bad habits? I can't see the logic in it for any kind of intentional phonetic stroke logic

Reply:
Totally happy to move it to bad habits. The only reason I put it in was because this outline was in Plover's dictionary, but not in any of yours, and I couldn't determine whether it was a "good" outline or not.